lang/scala-docs/Portfile

```diff
@@ -14,43 +15,39 @@ long_description    {*}${description}
 homepage             https://www.scala-lang.org/
 supported_archs      noarch
 
+# remove after 20230916
 if {${name} eq ${subport}} {
     PortGroup       stub 1.0
 
     version         0
-    revision        0
+    revision        1
+
+    replaced_by     scala
 }
 
 subport scala2.13-docs {
     version         2.13.8
-    revision        1
-    checksums       rmd160  f5a4fce12f94ec55ba3091868c234015eeb36a0e \
-                    sha256  7f99d5f0df20367b8efe85fdfada3e86cbc923cfee7a24274267e279e8350860 \
-                    size    63109932
+    revision        2
+    replaced_by     scala2.13
 }
 
 subport scala2.12-docs {
     version         2.12.16
-    revision        1
-    checksums       sha256  920b4ad8278eb287ffc2fd06c4623f878bde02f3babb37600c7bbf883b1ae4d7 \
-                    rmd160  3773ccc43c6135151933653382a54c48dc8b286b \
-                    size    57508160
+    revision        2
+    replaced_by     scala2.12
 }
 
 subport scala2.11-docs {
     version         2.11.12
-    revision        1
-    checksums       sha256  380f522e287308e8e61ca6ff7abfadf50525e15cd42e05fa84fae8caf07b39b3 \
-                    rmd160  82b5dd878d595ab9c9c6ea4660000313285dd8dc \
-                    size    48417340
+    revision        2
+    replaced_by     scala2.11
 }
 
 subport scala2.10-docs {
     version         2.10.7
-    revision        1
-    checksums       sha256  866a1fc287b4ac3e585b1b47ce59176ac3afff90c4543e106bc11ed8ff006d56 \
-                    rmd160  f0ca721ac3d9fb808b602a94c22806acfc576c19 \
-                    size    3412612}
+    revision        2
+    replaced_by     scala2.10
+}
 
 if {${name} ne ${subport}} {
```

Comment:
if this entire port is now obsolete, you'll want to move the inclusion of pg `stub` to the top level, and eliminate this entire block.

Reply:
> if this entire port is now obsolete, you'll want to move the inclusion of pg `stub` to the top level, and eliminate this entire block.

Ditto for removing the following, none of which are needed for an obsolete stub port:

```
license             Apache-2
maintainers         {@catap korins.ky:kirill} openmaintainer
description         The Scala Programming Language API docs
long_description    {*}${description}
homepage             https://www.scala-lang.org/
supported_archs      noarch
```